css/src/components/layout.scss

```diff
@@ -0,0 +1,114 @@
+@use 'sass:math';
+
+.layout {
+	display: flex;
+	flex-direction: column;
+	// min-inline-size: 100vh;
+	max-inline-size: 100vw;
+	overflow-x: hidden;
```

Comment:
Do we want both `overflow-x` and `overflow-inline`, or will just one suffice?

(Suspecting the answer is that `overflow-x` is a fallback for `overflow-inline`, but wanted to ask just in case)

Reply:
I removed it